math/primalityTest.go

```diff
@@ -0,0 +1,36 @@
+//A primality test is an algorithm for determining whether an input number is prime.Among other fields of mathematics, it is used for cryptography.
+//Unlike integer factorization, primality tests do not generally give prime factors, only stating whether the input number is prime or not.
+//Source - Wikipedia https://en.wikipedia.org/wiki/Primality_test
+package main
```

Comment:
`primalityTest` is not a good name for this prime checking implementation. Please rename it to `prime_check.go` and implement a test file also.